cubic-server/Client.hpp

```diff
@@ -65,6 +65,8 @@ class Client
     void sendPingResponse(int64_t payload);
     // Should I just pass the packet directly?
     void sendChatMessageResponse(const protocol::PlayerChatMessage &packet);
+    void disconnectClient(const chat::Message &message);
```

Comment:
And maybe also just call the function `disconnect` instead of `disconnectClient`, since the `Client` would be repetitive.
You could also give a default reason if none is passed like `You have been disconnected by the server` for example.